frontend/public/index.html

```diff
@@ -9,11 +9,7 @@
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```

Comment:
Better keep this and replace the corresponding file logo with your own. The comment belongs to the manifest which you should keep as well (unless you delete the logo files and favicon).